src/main/java/it/gov/pagopa/debtposition/controller/pd/actions/api/impl/DebtPositionActionsController.java

```diff
@@ -24,14 +25,20 @@ public class DebtPositionActionsController implements IDebtPositionActionsContro
     private ModelMapper modelMapper;
     @Autowired
     private PaymentPositionActionsService paymentPositionActionsService;
+    
+    @Value("${nav.aux.digit}")
+    private String auxDigit;
 
     @Override
     public ResponseEntity<PaymentPositionModel> publishDebtPosition(String organizationFiscalCode, String iupd) {
         log.info(String.format(LOG_BASE_HEADER_INFO, "POST", "publishDebtPosition", String.format(LOG_BASE_PARAMS_DETAIL, organizationFiscalCode, iupd)));
 
         PaymentPosition publishedDebtPos = paymentPositionActionsService.publish(organizationFiscalCode, iupd);
         if (null != publishedDebtPos) {
-            return new ResponseEntity<>(modelMapper.map(publishedDebtPos, PaymentPositionModel.class), HttpStatus.OK);
+        	PaymentPositionModel paymentPosition = modelMapper.map(publishedDebtPos, PaymentPositionModel.class);
+        	//PAGOPA-1155: add nav info to PO
+        	paymentPosition.getPaymentOption().forEach(po -> po.setNav(auxDigit+po.getIuv()));
```

Comment:
why not use `modelMapper` to set the nav?

Reply:
**modelmapper** is not a spring class. The **@value** annotation cannot be used